packages/react/src/Fabric/useAddresses.tsx

```diff
@@ -0,0 +1,25 @@
+import { Fabric } from "@signalwire/js";
+import { useEffect, useState } from "react";
+
+/**
+ * Given a client, poll the addresses available.
+ * @param client is the fabric client
+ * @param delay in ms, the polling time
+ * @returns null | address[]
+ */
+export function useAddresses(client: Fabric.Client | null, delay = 10000) {
```

Comment:
we should hardcode the delay (10,000 is fine) to make sure that the interface won't change once this is implemented with the proper apis